flake.nix

```diff
@@ -0,0 +1,88 @@
+{
+  inputs = {
+    nixpkgs.url = "github:NixOS/nixpkgs/nixos-unstable";
```

Comment:
Can you pin to the stable version? In the `flake.nix`, I see we use regular features, so it does not need to be unstable, right?

Reply:
Sure, we can use nixpkgs 24.05. I usually opt to use unstable because the lockfile locks the packages anyway (if I'm not wrong), so it's easier to have one thing to update rather than two.

Reply:
This seems like the right call actually, I'll change it.

Reply:
Done!